### PR TITLE
remove doc that encourages writing emulator.

### PR DESCRIPTION
**Pull request by pongad**

### CODE REVIEW

**Comment by pongad** [comment]
Update https://github.com/GoogleCloudPlatform/google-cloud-java/issues/1429

File: TESTING.md [hunk 1]
```diff
@@ -113,41 +112,6 @@ We recommend that you start the emulator on the remote machine using the [Google
 gcloud beta emulators datastore start --host-port <hostname of machine>:<port>
 ```
 
-### Testing code that uses DNS
```
Comment (pongad) [inline TESTING.md]: Done.

**Comment by pongad** [comment]
@garrettjonesgoogle  PTAL

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**